IOS/SwiftSeedProject/SwiftSeedProject/Domain/Models/User.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  User.swift
+//  SwiftSeedProject
+//
+//  Created by Brian Sztamfater on 4/4/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+struct User {
```

Comment:
User should be a class, mostly because when you pass a user instance you expect a reference, not a copy of that user.

That's an important difference between struct and class.